### ISSUE TITLE
Transaction fails on Private Chain

### ISSUE THREAD

**Issue by NeoParticle** [neo-project/docs#321]
Hi, I have created a private chain using NEO CLI, I got 100 M neo on the contract, I transferred those to one of the four wallets in the private chain. Now when I try to transfer some(eg. 100)neo to another wallet it fails. And also it is reduced from the wallet from which I transferred.

**Comment by NeoParticle** [comment]
1. Hi I did everything from beginning, now my all 4 wallets are newly created and consensus started. Now I tried to transfer 100 neo from contract address to first account - singned by 2nd and 3rd wallet - as a result I got all 100M transferred to the first wallet and not 100.
2. Now from the first wallet I transferred 100 NEO to 2nd Wallet - Failed - But Neo got reduced in the first wallet.
3. Now I transferred from the first wallet to the 4th Wallet - Sucess - So I created another wallet using GUI - transferred 100 NEO to the new one - Sucess
4. To be confirmed-  All values 100M will be transferred and cannot split the transfer, The wallets which are used to sign the contract cannot receive NEO. Why the failed transaction did not refund the NEO's.
